templates/react/components/foo/List.js

```diff
@@ -37,7 +38,10 @@ class List extends Component {
             <tr>
               <th>Id</th>
 {{#each fields}}
+  {{#contains name "id" }}
```

Comment:
Should be reverted IMO, the `id` field isn't the same thing than the `@id` property (in case of composed identifier for instance).